staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/validation_test.go

```diff
@@ -1849,6 +1849,75 @@ func TestValidationExpressions(t *testing.T) {
 	}
 }
 
+func TestCELValidationLimit(t *testing.T) {
+	tests := []struct {
+		name   string
+		schema *schema.Structural
+		obj    interface{}
+		valid  []string
+	}{
+		{
+			name:   "date format",
+			obj:    objs("1997-07-16", "1997-07-16"),
+			schema: schemas(dateFormat, dateFormat),
+			valid: []string{
+				ValsEqualThemselvesAndDataLiteral("self.val1", "self.val2", "timestamp('1997-07-16T00:00:00.000Z')"),
+				"self.val1.getDate() == 16",
+				"self.val1.getMonth() == 06", // zero based indexing
+				"self.val1.getFullYear() == 1997",
+				"type(self.val1) == google.protobuf.Timestamp",
```

Comment:
If we don't care what tests are used, maybe use something super simple instead of copying in one of the existing test sets?